esp32-srne-asf-example.yaml

```diff
@@ -772,13 +772,23 @@ switch:
   - platform: modbus_controller
     use_write_multiple: true
     modbus_controller_id: inverter0
-    name: "Alarms"
+    name: "Buzzer"
```

Comment:
I prefer `Audible fault alert` or `Acoustic fault notification` here.

Reply:
Alright. That's what this setting does indeed. Enables/disables the buzzer. I'll incorporate one of these.

Reply:
We could call it `Acoustic alarm notification` so it does match with the new feature.